### PR TITLE
Control panel

### PR DESCRIPTION
**Pull request by ONE001**

### CODE REVIEW

File: bin/server.py [hunk 1]
```diff
@@ -128,6 +133,7 @@ def __init__(self):
         handlers = [
             (r'/', IndexHandler),
             (r'/termsocket', TermSocketHandler),
+            (r'/control-panel', ControlPanelHandler),
```
Comment (eugulixes) [inline bin/server.py]: Substitute `/control-panel` for `/experimental` to show that the feature is not finished.

**Comment by dshil** [comment]
ping @ONE001, @eugulixes, 
There is an strange behavior: when I click on the full screen button, the command `rsz,24x80` appear in the console.

**Comment by eugulixes** [comment]
@dshil, it's ok. `rsz` used to be a command. So, this is the main reason why `control-panel` is experimental.

**Comment by dshil** [comment]
@eugulixes, ok, could we merge this PR with such behavior?

**Comment by eugulixes** [comment]
@dshil, yes.

**Review by dshil — APPROVED** [review]
LGTM

**Pull request merged**